test/tutorial/api_changes_from_dnf4/download_callbacks.cpp

```diff
@@ -0,0 +1,50 @@
+class PackageDownloadCallbacks : public libdnf5::repo::DownloadCallbacks {
```

Comment:
Please explicitly include all header files needed for all used types.

Reply:
Good point, I added them.